build_tools/github_actions/configure_ci.py

```diff
@@ -97,7 +100,11 @@ def write_job_summary(summary: str):
 
 
 def check_description_and_show_diff(original_description: str,
-                                    current_description: str):
+                                    original_labels: Sequence[str],
+                                    current_description: str,
+                                    current_labels: Sequence[str]):
+  original_description += "\n\nLabels:\n" + "\n".join(original_labels)
+  current_description += "\n\nLabels:\n" + "\n".join(current_labels)
   if original_description == current_description:
     return
```

Comment:
I think it's better to compare actual objects rather than a custom-formatted string. Also, does order matter?

```suggestion
  original_labels = sorted(original_labels)
  current_labels = sorted(current_labels)
  if original_description == current_description and original_labels == current_labels:
    return
  original_description += "\n\nLabels:\n" + "\n".join(original_labels)
  current_description += "\n\nLabels:\n" + "\n".join(current_labels)
```

I wonder if we shouldn't even go a step further and only compare the labels and trailers, showing the full diff only if those differ. If someone fixed a typo in their description, I don't think we care unless it affects those.

Reply:
Sort and change to compare labels separately

For PR description (title + body), I tried not to be too smart there. This is an informative warning about the changes on the latest PR description that may/may not affect the workflow, so raw data might be helpful for debugging? We can change to only compare trailers and labels if the warnings are too annoying.